### ISSUE TITLE
Missing documentation on switching regions

### ISSUE THREAD

**Issue by ArthurREGNARD** [firebase/firebase-functions-python#131]
Hello,

I'm migrating from JS to Python, but my functions are deployed in US and I'm in France/Europe. 
How can I set deployement region ? 

before in JS :
```code
import * as firebaseFunctions from "firebase-functions";

export const fonctions = firebaseFunctions.region("europe-west1");
```

Thinks for the lib,

**Comment by exaby73** [comment]
Hello @ArthurREGNARD. You can use the following code:

```python
from firebase_functions import options

options.set_global_options(region=options.SupportedRegion.EUROPE_WEST1)
```

Hope this helps you :) I will close this issue since this is not a bug or feature request. Thank you

**Comment by ArthurREGNARD** [comment]
@exaby73 ,

Yes that help me, but for someone else it's could be helpfull to put this informations in documentation ...

**Comment by exaby73** [comment]
Thanks for pointing that out @ArthurREGNARD. I'll reopen this issue to track this change :)

**Comment by exaby73** [comment]
Since the docs were updated, I will close this issue
